src/microbe_stage/components/MicrobeControl.cs

```diff
@@ -215,11 +215,31 @@ public static class MicrobeControlHelpers
         return true;
     }
 
+    /// <summary>
+    ///   Sets microbe speed straight forward.
+    /// </summary>
+    /// <param name="control">Control to hold commands.</param>
+    /// <param name="speed">Speed at which to move.</param>
     public static void SetMoveSpeed(this ref MicrobeControl control, float speed)
     {
         control.MovementDirection = new Vector3(0, 0, -speed);
     }
 
+    /// <summary>
+    ///   Moves microbe towards target position, even if that position is not forward.
+    ///   This does NOT handle any turning.
+    /// </summary>
+    /// <param name="control">Control to hold commands.</param>
+    /// <param name="selfPosition">Position of microbe moving.</param>
+    /// <param name="targetPosition">Vector3 that microbe will move towards.</param>
+    /// <param name="speed">Speed at which to move.</param>
+    public static void SetMoveSpeedPrecise(this ref MicrobeControl control, WorldPosition selfPosition,
+        Vector3 targetPosition, float speed)
+    {
+        control.MovementDirection = selfPosition.Rotation.Inverse()
+            * (targetPosition - selfPosition.Position) * speed;
```

Comment:
What happens if the vector is 0 length here (same position currently as target)? I worry that it will cause NaNs or other problems. For safety I would do something like this:
```c#
var vectorToTarget = targetPosition - selfPosition.Position;
if (vectorToTarget.LenqthSquared() < MathUtils.EPSILON)
{
    control.MovementDirection = Vector3.Zero;
    return;
}
```